package.json

```diff
@@ -13,10 +13,11 @@
     "test": "jest",
     "test:watch": "jest --watch",
     "test:coverage": "jest; open ./coverage/lcov-report/index.html",
-    "prisma:pull": "dotenv -e .env.local npx prisma db pull",
+    "prisma:pull": "dotenv -e .env.local -- npx prisma db pull",
     "prisma:generate": "prisma generate",
-    "prisma:migrate": "dotenv -e .env.local npx prisma migrate dev",
-    "prisma:studio": "dotenv -e .env.local npx prisma studio",
+    "prisma:migrate": "dotenv -e .env.local -- npx prisma migrate dev",
+    "prisma:migrate-create-only": "dotenv -e .env.local -- npx prisma migrate dev --create-only",
```

Comment:
Spent way too long trying to get the `--create-only` flag to be picked up here. 

Classic case of RTFM 🤦 - explainer in docs here why the additional `--` is required https://www.prisma.io/docs/orm/more/development-environment/environment-variables/managing-env-files-and-setting-variables#using-dotenv-cli-via-command-line